src/components/tooltip/TooltipDirective.js

```diff
@@ -239,6 +246,16 @@
                     feature.set('htmlpopup', htmlpopup);
                     showFeatures(layerToQuery.getExtent(), size,
                         [feature]);
+                    // Iframe communication from inside out
+                    if (top != window) {
+                      var featureId = feature.getId();
+                      var layerBodId = layerToQuery.get('bodId');
+                      if (featureId && layerBodId) {
```

Comment:
Not sure

Where the window.parent property returns the immediate parent of the current window, window.top returns the topmost window in the hierarchy of window objects.

And that's what we want no?